Gruntfile.js

```diff
@@ -8,9 +8,7 @@ module.exports = function (grunt) {
   require('time-grunt')(grunt);
 
   // load all grunt tasks
-  require('load-grunt-tasks')(grunt, {
-    pattern: ['grunt-*', '!grunt-template-jasmine-istanbul']
-  });
+  require('jit-grunt')(grunt, {});
```

Comment:
`jit-grunt` is great, but a potential gotcha - if the npm module is named differently from the grunt task then you have to give it the mapping. I actually think that's not a problem for your current tasks, but if you add e.g. `grunt-npm` then you'd need a line like this: https://github.com/freedomjs/freedom-pgp-e2e/blob/master/Gruntfile.js#L15